packages/providers/src/json-rpc-provider.ts

```diff
@@ -131,7 +131,7 @@ export class JsonRpcProvider extends Provider {
      * @returns {Promise<FinalExecutionOutcome>}
      */
     async sendTransactionAsync(signedTransaction: SignedTransaction): Promise<FinalExecutionOutcome> {
-        return this.sendTransactionUntil(signedTransaction, 'NONE');
+        return this.sendTransactionUntil(signedTransaction, 'INCLUDED');
```

Comment:
I think we can leave it as `NONE`, since anyway people will have to check its status, and that way we don't lose time on the overhead of waiting for the RPC to answer